public/app/plugins/datasource/loki/components/useLokiLabels.ts

```diff
@@ -39,7 +39,7 @@ export const useLokiLabels = (
   };
 
   const tryLabelsRefresh = async () => {
-    await languageProvider.refreshLogLabels(absoluteRange);
+    await languageProvider.refreshLogLabels(absoluteRange, true);
```

Comment:
We are setting `forceRefresh = true` and therefore we are fetching labels even though we didn't wait for 30s ( LABEL_REFRESH_INTERVAL.  We always want to have labels and values that are up to date with time range.

Reply:
Do we have to use `force = true` here? Ideally condition inside would check if the range is different and invalidate the refresh interval. If the range does not change than we should probably still adhere to that interval so we do not spam the server unnecessarily.